src/main/scala/co/datamechanics/delight/Configs.scala

```diff
@@ -57,6 +57,6 @@ object Configs {
     s"$sanitizedAppName-$uuid"
   }
 
-  val delightVersion: String = "1.0.2"
+  val delightVersion: String = "1.0.3"
```

Comment:
Why is this not a part of build (to generate a version file at build time)?

Reply:
What about using a version range? https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Ivy+revisions?

Even in the Docker world where `latest` has always been used they seem to avoid this version uncertainty to tag an image with two tags - a version and `latest`.

> the user never have to worry about upgrading Delight themselves

Shouldn't the user be aware of this version change if it happens? It could impact the user's deployments, no?

One way to work it out would be to have the version in a separate text version file and load it at runtime, e.g.

```
// pseudo code
val delightVersion: String = this.getClass.getClassLoader.getResourceAsStream("META-INF/version")
```

Perhaps one day that triggers a better solution (once it's outside Scala space)?

Reply:
You're right. I'll implement the bumping of a version file in a different PR and import the version number from the code.

> Shouldn't the user be aware of this version change if it happens? It could impact the user's deployments, no?

In this case, we favor simplicity over debugging.